bin/pt-kill

```diff
@@ -7373,20 +7373,46 @@ sub main {
 
          MATCHING_QUERY:
          foreach my $query ( @queries ) {
+            my $fp = $qr->fingerprint($query->{'Info'});
+            my $chksm = Transformers::make_checksum($fp);
```

Comment:
I've moved this outside of the `if ( $o->get('query-id') ) ...` scope so that it can be used within the `if ( $o->get('log-json') ) ...` scope. It is now always executed, but only output when `--query-id` is specified (as before), or in the JSON output when `--log-json` is specified.